pkg/restore/restore.go

```diff
@@ -993,6 +1009,7 @@ func (ctx *restoreContext) restoreItem(obj *unstructured.Unstructured, groupReso
 	}
 
 	if groupResource == kuberesource.PersistentVolumes {
+		resetStatus(obj)
```

Comment:
This line seems not necessary as this code block doesn't really create the object. And there is one call resetMetadataAndStatus at line 1133 later before creating the object

Reply:
Sorry for any misleading, but by the last comment I meant the line at 1012, resetStatus(obj), is not necessary, so please consider and remove it. Thanks.